src/v2/components/Channel/Channel.tsx

```diff
@@ -595,8 +595,8 @@ export const ChannelWithContext = <
     Re,
     Us
   >['loadChannelAtMessage'] = ({ after = 2, before = 10, messageId }) =>
-    channelQueryCall(() => {
-      queryAtMessage({ after, before, messageId });
+    channelQueryCall(async () => {
+      await queryAtMessage({ after, before, messageId });
```

Comment:
was something failing with this not being async?

Reply:
Yeah, we wait for this call inside channelQueryCall, and once it is complete, we set loading state as false.  Without await here, loading state will be set to false even before query is complete